File: OOP/Aula_07-Herenca/ContaCorrente.java

public class ContaCorrente extends Conta {

	@Override
	public void depositar ( double deposito ) {

		deposito -= 0.10; // Taxa de deposito
		this.setSaldo (this.getSaldo() + deposito);
	}
	
	@Override
	public void atualizar ( double taxa ) {

		taxa *= 2;
		this.setSaldo( this.getSaldo() + (this.getSaldo() * taxa) );
	}

}

File: OOP/Aula_08_Classes_Abstratas/Att_01-05/Entities/ContaCorrente.cs
namespace Entities
{
	public class ContaCorrente : Conta
	{
		public override void Depositar(double deposito)
		{
			deposito -= 0.10;
			saldo += deposito;
		}
		public override void Atualizar(double taxa)
		{
			taxa *= 2;
			saldo += saldo * taxa;
		}
	}
}
